distribution/ddf-common/src/main/resources/etc/org.apache.karaf.features.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<!--
+    This file was manually created for right now.
+-->
+<featuresProcessing xmlns="http://karaf.apache.org/xmlns/features-processing/v1.0.0"
+                    xmlns:f="http://karaf.apache.org/xmlns/features/v1.5.0">
+
+    <!-- A list of blacklisted features XML repository URIs - they can't be added later -->
+    <blacklistedRepositories/>
+
+    <!-- A list of blacklisted feature identifiers that can't be installed in Karaf and are not part of the distribution -->
+    <blacklistedFeatures>
+        <feature>pax-jetty-http2</feature>
+        <feature version="[9.3,9.4.10]">pax-jetty</feature>
+        <feature version="[7.0.0,7.2.2]">pax-http-jetty</feature>
+        <feature version="[7.0.0,7.2.2]">pax-http</feature>
+        <feature version="[7.0.0,7.2.2]">pax-war</feature>
+        <feature version="[7.0.0,7.2.2]">pax-http-tomcat</feature>
+        <feature version="[7.0.0,7.2.2]">pax-war-tomcat</feature>
+        <feature version="[7.0.0,7.2.2]">pax-jsf-support</feature>
+        <feature version="[7.0.0,7.2.2]">pax-jsf-resources-support</feature>
+        <feature version="[7.0.0,7.2.2]">pax-http-undertow</feature>
+        <feature version="[5.0.4.RELEASE_1,5.0.4.RELEASE_1]">spring</feature>
```

Comment:
Looks like this will have a conflict with https://github.com/codice/ddf/pull/3783/files.